h2o-algos/src/main/java/hex/schemas/AdaBoostV3.java

```diff
@@ -0,0 +1,41 @@
+package hex.schemas;
+
+import hex.adaboost.AdaBoost;
+import hex.adaboost.AdaBoostModel;
+import water.api.API;
+import water.api.schemas3.ModelParametersSchemaV3;
+
+public class AdaBoostV3 extends ModelBuilderSchema<
+        AdaBoost,
+        AdaBoostV3, 
+        AdaBoostV3.AdaBoostParametersV3> {
+
+    public static final class AdaBoostParametersV3 extends ModelParametersSchemaV3<AdaBoostModel.AdaBoostParameters, AdaBoostParametersV3> {
+        static public String[] fields = new String[]{
+                "model_id",
+                "training_frame",
+                "ignored_columns",
+                "ignore_const_cols",
+                "categorical_encoding",
+                "weights_column",
+
+                // AdaBoost specific
+                "nlearners",
+                "weak_learner",
+                "learn_rate",
+                "seed",
+        };
+
+        @API(help = "Number of AdaBoost weak learners.", gridable = true)
+        public int nlearners;
+
+        @API(help = "Choose a weak learner type. Defaults to DRF.", gridable = true, values = {"AUTO", "DRF", "GLM", "GBM"})
+        public AdaBoostModel.Algorithm weak_learner;
```

Comment:
Any reason why deeplearning is not included?

Reply:
No, I will do it as a part of [#15779](https://github.com/h2oai/h2o-3/issues/15779) thanks